### ISSUE TITLE
Enabling the Script on Windows

### ISSUE THREAD

**Issue by Jochen1980** [laroche/tws-api-examples#3]
I played around and it seems to me that Windows can cause some pain in the ass here. I tried to adjust the file names myself but I had no success, yet. Anyway, I guess 95 % of the users are running their TWS in a Windows Desktop environment, so maybe we should spend the effort to make this tool available on linux and windows.

**Comment by laroche** [comment]
Yes, hope this will also run on Windows. I've added some more check for opening the database, so maybe
for your case this now gives a better error message.

Possible problem might be that the subdirectory "data" does not exist, then also the database file cannot be
created.

Does the problem still exist with the current version? Is an error message reported?

best regards,

Florian La Roche
